
(checker.py:1430): dbind-WARNING **: 13:35:12.923: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-EfqdA8FD5Y: Connection refused
